bin/improver-tests

```diff
@@ -224,25 +198,6 @@ Arguments:
                     Valid names are: pycodestyle, pylint, pylintE, licence,
                     doc, unit, cli. pycodestyle, pylintE, licence, doc, unit,
                     and cli are the default tests.
```

Comment:
Losing the ability to run a single test/group of tests (through the `improver` wrapper) is a bit of a shame - but given that we can just run `pytest` directly, I don't think this is a major loss. Especially when balanced with simplifying this code and the other benefits that `pytest` brings.

Reply:
We could re-add this functionality in the future by passing through any other args provided to the `improver tests` wrapper onto pytest, such as via `"$@"` in shell script.

Reply:
(If we need it at a later date, it shouldn't be difficult to add, yep)

Reply:
We need to port that shell script to python, this could be addressed then.